### ISSUE TITLE
HDP installation hangs at 8%

### ISSUE THREAD

**Issue by akanto** [sequenceiq/cloudbreak#1461]
The installation of HDP hangs at 8%. The Ambari logs shows that yum install hadoop_2_4_* never returns:

```
2016-04-05 18:42:16,677 - File['/etc/yum.repos.d/HDP-UTILS.repo'] {'content': InlineTemplate(...)}
2016-04-05 18:42:16,678 - Writing File['/etc/yum.repos.d/HDP-UTILS.repo'] because it doesn't exist
2016-04-05 18:42:16,678 - Package['unzip'] {}
2016-04-05 18:42:17,135 - Skipping installation of existing package unzip
2016-04-05 18:42:17,135 - Package['curl'] {}
2016-04-05 18:42:17,170 - Skipping installation of existing package curl
2016-04-05 18:42:17,170 - Package['hdp-select'] {}
2016-04-05 18:42:17,202 - Installing package hdp-select ('/usr/bin/yum -d 0 -e 0 -y install hdp-select')
2016-04-05 18:42:26,405 - Package['hadoop_2_4_*-yarn'] {}
2016-04-05 18:42:26,493 - Installing package hadoop_2_4_*-yarn ('/usr/bin/yum -d 0 -e 0 -y install 'hadoop_2_4_*-yarn'')
```

**Comment by saurabhmishra** [comment]
We where able to proceed further from here by retrying couple of times but then it will get stuck at mysql install for hive metastore

**Comment by akanto** [comment]
On 31st March 2016 the Systemd package has been updated with a newer version in the central CentOS repository which caused incompatibilities during HDP install and that is the reason why the cluster stopped at 8%. The main problem is that when the Ambari installs the `rpcbind` package then the Systemd also gets updated and when the next package which had systemd service file packaged with it is installed the yum is completely freeze.

We are working on the fix.

**Comment by martonsereg** [comment]
We have built a new Cloudbreak container for a workaround until we come out with a proper patched cbd. To use it, please include this line in your cbd `Profile`:
```
export DOCKER_TAG_CLOUDBREAK=1.2.4-rc.1
```
Save the file and use these commands to generate a new cbd configuration and to restart Cloudbreak: 
```
cbd kill
cbd regenerate
cbd start
```

**Comment by saurabhmishra** [comment]
Able to spin up cluster using the above fix using custom blueprint and custom host templates in aws.

Thank You.

**Comment by doktoric** [comment]
This is already fixed in 1.2.1
